src/common/FileSystem.cpp

```diff
@@ -1142,10 +1142,14 @@ static void InternalLoadPak(const PakInfo& pak, Util::optional<uint32_t> expecte
 	// Save the real checksum in the list of loaded paks (empty for directories, not used for legacy paks)
 	loadedPaks.back().realChecksum = realChecksum;
 
-	// Get the timestamp of the pak
-	loadedPaks.back().timestamp = FS::RawPath::FileTimestamp(pak.path, err);
-	if (err)
-		return;
+	// Get the timestamp of the pak, but only for directories. Directories don't need timestamp.
```

Comment:
Comment says "only for directories" when the opposite is intended.

Reply:
What is the timestamp used for?

Reply:
The timestamp is not used, except in the implementation of a VM syscall to return the timestamp, which is also not used. Maybe the intention was to eventually avoid reloading paks that are already in memory and not modified, or something.